doc/03_reference/verification.rst

```diff
@@ -89,20 +89,15 @@ Prerequisites & Environment Setup
 
 In order to run the co-simulation flow, you'll need:
 
-- A SystemVerilog simulator that supports UVM. The flow is currently
-  tested with VCS.
+- A SystemVerilog simulator that supports UVM. The flow is currently tested with VCS.
 
 - A RISC-V instruction set simulator, such as Spike_ or OVPsim_.
-  Note that Spike must be configured with
-  ``--enable-commitlog`` and ``--enable-misaligned``. The commit log
-  is needed to track the instructions that were executed and
-  ``--enable-misaligned`` tells Spike to simulate a core that
-  handles misaligned accesses in hardware (rather than jumping to a
-  trap handler).
-  In addition, Spike does not support the `RISC-V Bit Manipulation Extension <bitmanip_>`_  (Bitmanip) by default.
-  To support this draft extension implemented in Ibex, the `riscv-bitmanip branch <Spike_>`_ of Spike needs to be used.
-  If it is desired to simulate the core with the Icache enabled, a lowRISC-specific branch
-  of Spike must be used, `found here <https://github.com/lowRISC/riscv-isa-sim/tree/ibex>`_.
+  Note that when building Spike the ``--enable-commitlog`` and ``--enable-misaligned`` options must be passed to the ``configure`` script.
+  ``--enable-commitlog`` is needed to produce log output to track the instructions that were executed.
+  ``--enable-misaligned`` tells Spike to simulate a core that handles misaligned accesses in hardware (rather than jumping to a trap handler).
+  If it is desired to simulate the core with the Icache enabled, a lowRISC-specific branch of Spike must be used, `found here <https://github.com/lowRISC/riscv-isa-sim/tree/ibex>`_.
+  Ibex supports v0.92 of the Bitmanip specification, the master branch of Spike_ and OVPSim_ may support a different version.
+  It is recommended the `lowRISC-specific branch of Spike <https://github.com/lowRISC/riscv-isa-sim/tree/ibex>`_ is used when using a configuration with Bitmanip to ensure the simulated version of the Bitmanip specification matches with the RTL implemented version.
```

Comment:
I wonder whether we should only link to this in one of the sentences? As a reader, I find it quite confusing when there are two links (with different link texts) that both go to the same place. There's a danger of the whole "but which one should I use?" question. Alternatively, get rid of "found here" in the first example and hyperlink the same text in both cases.

Reply:
I think we want the link in both places, I'll do as you suggest and alter it so links have the same text (I think we might want to avoid links like 'found here' anyway preferring to just link the text that directly names whatever you're linking to anyway).